package.json

```diff
@@ -1,25 +1,23 @@
 {
   "name": "simple-app",
   "version": "1.0.0",
-  "description": "Simple App Plugin",
-  "author": "Grafana Labs",
-  "license": "Apache-2.0",
+  "description": "Grafana App Plugin Template",
   "scripts": {
     "build": "grafana-toolkit plugin:build",
     "test": "grafana-toolkit plugin:test",
     "dev": "grafana-toolkit plugin:dev",
     "watch": "grafana-toolkit plugin:dev --watch",
-    "start": "yarn watch",
-    "test:accessibility": "pa11y-ci --config .pa11yci.json"
-  },
-  "dependencies": {
-    "@grafana/data": "next",
-    "@grafana/runtime": "next",
-    "@grafana/ui": "next"
+    "sign": "grafana-toolkit plugin:sign"
   },
+  "author": "Grafana Labs",
+  "license": "Apache-2.0",
   "devDependencies": {
-    "@grafana/toolkit": "next",
-    "eslint-plugin-react-hooks": "4.0.8",
-    "pa11y-ci": "^2.4.0"
+    "@grafana/data": "latest",
+    "@grafana/runtime": "latest",
+    "@grafana/ui": "latest",
```

Comment:
Given that `data`, `runtime` and `ui` all end up in the bundle (well, at least `runtime` and `ui`, not sure about what `data` is used for off the top of my head), how come that you moved them to `devDependencies`?

Reply:
Syncing with the other starters here. Not an expert here, but I believe it's to avoid including them twice, since Grafana bundles these. Same thing with dependencies like react and emotion etc.

Reply:
But we don't have react and emotion here? So, based on that, `data`, `runtime` and `ui` should be removed completely.
Which... would probably be a good idea **if** building still works. :)

Reply:
Sorry for the confusion, just wanted to give a couple of examples of similar situations :)

If you don't add them to devDependencies at least, you won't have access to the types, so wouldn't it be difficult to develop?

Reply:
Well, again, the same argument would hold true for react too. Obviously, one can use react types just fine, even though it's not a direct dependency.

Reply:
Oh, you mean that the plugin isn't even using the packages? Then yes, we can totally remove them.